# ui_pqExportStateWizard.h is only generated if Python part is build
# http://www.paraview.org/Bug/bug_relationship_graph.php?bug_id=15655&graph=relation
--- ParaView-v4.4.0-source/Qt/Components/CMakeLists.txt.orig	2015-09-11 19:58:50.000000000 +0200
+++ ParaView-v4.4.0-source/Qt/Components/CMakeLists.txt	2015-10-09 19:54:48.185948605 +0200
@@ -640,7 +640,7 @@
 #the pqSGExportStateWizard has subclasses that directly access
 #the UI file, and currently we don't have a clean way to break this hard
 #dependency, so for no we install this ui file.
-if(PARAVIEW_INSTALL_DEVELOPMENT_FILES)
+if(PARAVIEW_INSTALL_DEVELOPMENT_FILES AND PARAVIEW_ENABLE_PYTHON)
   install(FILES "${CMAKE_CURRENT_BINARY_DIR}/ui_pqExportStateWizard.h"
           DESTINATION "${VTK_INSTALL_INCLUDE_DIR}")
 endif()
